docs/src/test/scala/docs/http/scaladsl/server/directives/DebuggingDirectivesExamplesSpec.scala

```diff
@@ -130,7 +130,7 @@ class DebuggingDirectivesExamplesSpec extends RoutingSpec {
     }
     def printResponseTime(log: LoggingAdapter) = {
       val requestTimestamp = System.nanoTime
-      akkaResponseTimeLoggingFunction(log, requestTimestamp)(_)
+      akkaResponseTimeLoggingFunction(log, requestTimestamp) _
```

Comment:
Hmm does this also work on 2.12.8? seemed this was changed intentionally at https://github.com/akka/akka-http/pull/2305/files#diff-d4736e778f9a72c660d7563af6fd6f6a

Reply:
I had to change it to this to make it compile on 2.12.8

Reply:
mmm that is weird... It was a warning and the builds passed...

Reply:
That diff does the same as this?

Reply:
Oooopssss I think coffee really didn't kick in yet.... It's the same change indeed

Reply:
oh huh indeed

Reply:
ah, this is the backport, sorry...